yajsapi/executor/index.ts

```diff
@@ -598,7 +598,7 @@ export class Executor {
             emit(evt);
             results.push(evt);
             if (evt instanceof events.CommandExecuted && !evt.success) {
-              throw new CommandExecutionError(evt.command, evt.message)
+              throw new CommandExecutionError(evt.command, evt.stderr)
```

Comment:
In `yapapi`, `CommandExecutionError` now has both `message` and `stderr` fields. You may want to change it also in the JS version.

Reply:
If not, I'd suggest leaving `evt.message` as the second argument of the constructor.

Reply:
Fixed